lxd/device/disk.go

```diff
@@ -114,6 +116,10 @@ func (d *disk) validateConfig() error {
 		return fmt.Errorf("The recursive option is only supported for additional bind-mounted paths")
 	}
 
+	if !strings.HasPrefix(d.config["source"], "ceph") && (d.config["ceph.cluster_name"] != "" || d.config["ceph.user_name"] != "") {
```

Comment:
Still needs updating for `ceph:` and `cephfs:`

Reply:
Everytime we do a merge it seems like our previous changes get overriden. Ill fix that quickly